schema.graphql

```diff
@@ -433,14 +447,85 @@ type Match {
 }
 
 """
-Docs group: Match
+Docs group: Question
+"""
+type Packet {
+  """
+  A human-readable identifier. Often primarily numeric (e.g., `Packet 1`), but could also be identified by authors (`Chicago` or `Yale + Maryland`), as for packet-submission tournaments.
+  """
+  identifier: String!
```

Comment:
Dare we call this `name` instead for the sake of consistency with other types?

Reply:
For some reason that doesn't feel quite right, but I'm okay with it.